src/aashay/ConditionalStatement/MultipleBankAccounts.java

```diff
@@ -0,0 +1,265 @@
+package aashay.ConditionalStatement;
+/*
+ * Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI
+
+ */
+
+import java.util.Scanner;
+
+public class MultipleBankAccounts {
+	double iciciBalance = 5000;
+	double sbiBalance = 5000;
+	double hdfcBalance = 5000;
+	double bobBalance = 5000;
+	
+	int creditCountICICI,creditCountSBI,creditCountHDFC,creditCountBOB,debitCountICICI,debitCountSBI,debitCountHDFC,debitCountBOB;
+	
+	void accntHandling( double amount, String operation, String bank) {
+		
+		switch(operation) {
+		case "Credit":
+			if(bank.equals("ICICI")) {
+					iciciBalance = iciciBalance + amount;
+					System.out.println("Credited amount "+amount+" in to your ICICI bank account. Your current balane is "+iciciBalance);
+					creditCountICICI++;
+			}else if(bank.equals("SBI")) {
+					sbiBalance = sbiBalance + amount;
+					System.out.println("Credited amount "+amount+" in to your SBI bank account. Your current balane is "+sbiBalance);
+					creditCountSBI++;
+			}else if(bank.equals("HDFC")) {
+				bobBalance = hdfcBalance + amount;
+					System.out.println("Credited amount "+amount+" in to your HDFC bank account. Your current balane is "+bobBalance);
+					creditCountHDFC++;
+			}else if(bank.equals("BOB")) {
+					bobBalance = bobBalance + amount;
+					System.out.println("Credited amount "+amount+" in to your BOB bank account. Your current balane is "+bobBalance);
+					creditCountBOB++;
+			}
+				break;
+		
+		case "Debit": 
+			if(bank.equals("ICICI")) {
+					if(amount <= iciciBalance ) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)